applications/tari_app_grpc/proto/wallet.proto

```diff
@@ -57,6 +62,29 @@ message TransferResult {
     string failure_message = 4;
 }
 
+message GetTransactionInfoRequest {
+    repeated uint64 transaction_ids = 1;
+}
+
+message GetTransactionInfoResponse {
+    repeated TransactionInfo statuses = 1;
+}
+
+message TransactionInfo {
+    uint64 tx_id = 1;
+    string address = 2;
+    enum Status {
+        STATUS_PENDING = 0;
+        STATUS_CANCELLED = 1;
+        STATUS_COMPLETE = 2;
+        STATUS_MINED = 3;
+    }
+    Status status = 3;
+    uint64 amount = 4;
+    uint64 fee = 5;
+    string transactionHash = 6;
```

Comment:
```suggestion
    bytes transaction_hash = 6;
```

Reply:
Same argument as for address. If we want a user to be able to query a transaction hash within the pool db, then it simplifies things to have a canonical string representation.

Reply:
Will change